extension/index.azureFunction.js

```diff
@@ -0,0 +1,49 @@
+import utils from './src/utils.js'
+import paymentHandler from './src/paymentHandler/payment-handler.js'
+import { getAuthorizationRequestHeader } from './src/validator/authentication.js'
+
+const FAILED_VALIDATION = 'FailedValidation'
+
+function handleResponse(context, paymentResult) {
```

Comment:
Currently you have handleResponse that actually handles also error responses and you have `handleErrorResponse` separately. 
Better use this method only for `handleSuccessResponse`. Then you would have `handleSuccessResponse` and `handleErrorResponse` which makes more sense.

The decision which response it is should be in the method `azureExtensionTrigger` like already have now. It doesn't make sense to have this decision both in `azureExtensionTrigger` and in `handleResponse`. So sth like this in `azureExtensionTrigger`:
```
if (paymentResult.actions)
    handleSuccessResponse()
else
    handleErrorResponse()
```

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/993/commits/b585915c2e020a647de58ae5551a0f52c997fd95